src/main/java/io/jenkins/plugins/ml/IPythonBuilder.java

```diff
@@ -54,18 +55,15 @@
     private final String code;
     private final String filePath;
     private final String parserType;
+    private FileExtension ext;
```

Comment:
Couldn't it be declared in the perform method? I mean, it does not seem to belong to the object.

Reply:
Yeah sure make it up within the method